app/components/case-studies-components/SimilarCaseStudies.tsx

```diff
@@ -1,45 +1,60 @@
 "use client";
 
+import { useState } from "react";
```

Comment:
Please remove unnecessary imports. :)

Reply:
whoops!

Reply:
Actually I'm using useState in this component. SHouldn't I keep it?

Reply:
Apologies for overlooking this. Great job!